T/t8code/build_tarballs.jl

```diff
@@ -6,19 +6,18 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "t8code"
-version = v"1.1.2"
+version = v"1.6.1"
 
-# Collection of sources required to complete build
-sources = [
-    ArchiveSource("https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8code_v$(version).tar.gz",
-                  "8a30206a8fb47013b3dafe7565cf8e09023df8373c1049e7e231d9fd36b011e4"),
+tarball = "https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8-$(version).tar.gz"
+sha256sum = "dc96effa7c1ad1d50437fefdd0963f6ef7c943eb10a372a4e8546a5f2970a412"
 
-    DirectorySource("./bundled")
-]
+# Collection of sources required to complete build
+sources = [ArchiveSource(tarball, sha256sum), DirectorySource("./bundled")]
 
 # Bash recipe for building across all platforms
-script = raw"""
-cd $WORKSPACE/srcdir/t8code
+script = """
+cd \$WORKSPACE/srcdir/t8-$(version)
+""" * raw"""
```

Comment:
We usually do something like
```suggestion
script = raw"""
cd $WORKSPACE/srcdir/t8*
```
which would work with any version